test/editor/Store/Helpers.re

```diff
@@ -0,0 +1,8 @@
+module Model = Oni_Model;
+
+let onStateChange = (updater, initialState, action, onChange) => {
+  let (storeDispatch, _) =
+    Isolinear.Store.create(~initialState, ~updater, ());
+
+  action |> storeDispatch |> fst |> onChange;
```

Comment:
Awesome! I like the way you abstracted this out to make it easy to reuse across tests.